robottelo/decorators/host.py

```diff
@@ -1,56 +1,35 @@
+"""Implements decorator regarding satellite host"""
 import logging
 import re
-import six
 import unittest2
 
+from backports.functools_lru_cache import lru_cache
```

Comment:
In fact I used that one because when I've tried to import lru_cache, this lib was already used on our project. Probably is dependency of another lib already, so it would not include another lib on our system. @rochacbruno